command/src/command.proto

```diff
@@ -745,7 +745,7 @@ message ProtobufAccessLog {
     // round trip time for the backend (microseconds)
     optional uint64 server_rtt = 10;
     // time for the backend to respond (microseconds)
-    required uint64 response_time = 12;
+    optional uint64 response_time = 12;
```

Comment:
Should be move at the position 21 and removed from the 12. Keep it optional :) Like that we keep the ability to use an old client on newer version

Reply:
done